### DISCUSSION TITLE
Predict Time-To-Event

### DISCUSSION THREAD

**Discussion by elopezfune** [sebp/scikit-survival#153] [Q&A]
I am working on several models derived from this library. 
I am trying to implement the calibration plots after training some models like Random Survival Forest, Componentwise Gradient Boosting, Survival Trees and Cox-Net; however, not all of them have implemented the predictions of the time-to-event, but they predict risk scores.
How could I read the predicted time to events? Is there a way to get them from the risk scores? 
There are not enough examples to guide the users!

**Comment by sebp** [comment]
Predictions are almost never the actual time of an event. Some models predict a survival function (`predict_survival_function`), which give you the probability of an event at time t or later. Have you had a look at the [user guide](https://scikit-survival.readthedocs.io/en/latest/user_guide/understanding_predictions.html) already?
